### ISSUE TITLE
Real numeric constants aren't colored properly

### ISSUE THREAD

**Issue by mkruselj** [nojanath/SublimeKSP#44]
Anything after the decimal point isn't syntax colored. To fix that, regexp for numerical constant needs to be updated from this:

`match: '(\s+-|(?<=[=,])-)?((0x)[0-9a-fA-F]*|[0-9][0-9a-fA-F]*[hH]|[0-9]+)\b'`

to this:

`match: '(\s+-|(?<=[=,])-)?((0x)[0-9a-fA-F]*|[0-9][0-9a-fA-F]*[hH]|[0-9.]+)\b'`

**Comment by mkruselj** [comment]
This actually removes syntax coloring from dots in .and., .or. and .not., so to fix that, match for operators needs to go FIRST, before punctuation. Like so:

```
scope: source.ksp
contexts:
  main:
    - match: '\band\b|\bor\b|\bxor\b|\bnot\b|[.]and[.]|[.]or[.]|[.]xor[.]|[.]not[.]|\bmod\b'
      comment: Operator
      scope: keyword.operator.source.ksp
    - match: "[,.()\\[\\]]"
      comment: Punctuation and brackets
      scope: punctuation
```

**Comment by neblix** [comment]
You provided your coloring fixes in pull request #66. Closing!
